eo-maven-plugin/src/it/custom_goals/README.md

```diff
@@ -0,0 +1,44 @@
+<img alt="logo" src="https://www.objectionary.com/cactus.svg" height="100px" />
+
+The purpose of `custom-goals` integration test is to check partial compilation
+of EO programs. We have to be sure that the plugin correctly performs particular
+set of goals in order to compile a program. We just check that for a
+particular set of goals the particular set of artifacts and folders will be
+compiled and create. The idea to use only specific goals of the plugin rather
```

Comment:
@volodya-lombrozo "...and created...",   "...the idea is.."

Reply:
@mximp Thanks. Fixed.